spec.bs

```diff
@@ -2832,15 +2839,21 @@ threshold when responding to [=query k-anonymity count=].
 </div>
 
 <div algorithm>
-  To <dfn>query ad k-anonymity count</dfn> given an [=interest group=] |ig| and a [=URL=] |ad|:
-    1. Let |keyString| be the [=string/concatenation=] of the following strings separated with U+000A LF:
+  To <dfn>query k-anonymity cache</dfn> given a |hashCode|:

```

Comment:
Can you specify the type of the `|hashCode|` parameter here?

Reply:
Done